packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/scale/index.js

```diff
@@ -3,18 +3,18 @@ import '@uirouter/angularjs';
 import 'oclazyload';
 
 const moduleName =
-  'ovhManagerPciProjectKubernetesDetailsNodesBulkDeleteLazyloading';
+  'ovhManagerPciProjectKubernetesDetailsNodePoolsScaleLazyloading';
 
 angular.module(moduleName, ['ui.router', 'oc.lazyLoad']).config(
   /* @ngInject */ ($stateProvider) => {
     $stateProvider.state(
-      'pci.projects.project.kubernetes.details.nodepools.details.nodes.bulk-delete.**',
```

Comment:
We completely remove the feature ?

Reply:
Yes, it can be done via the new scale component.

Reply:
I wonder if we should add a redirection in this case then

Reply:
I am not sure to get your point @marie-j.

Reply:
Well if I'm a customer and I access to the `/delete` url directly I will no longer be able to so I wonder if it is relevant to add a redirection in that case or not

Reply:
I am not sure it's relevant.
1 - The logic is to navigate from the UI to do things.
2 - To delete nodes our customer must use OVH API note using a saved manager URL.
3 - If it do it via the URL at least he should save one or many urls for eache nodes he had.
4 - If we do i special page to cover the redirection we must planned a new global feature to cover this case. (may be we have this @marie-j ?)